docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -571,6 +571,21 @@ the state of the table to a previous snapshot id::
 
     CALL iceberg.system.rollback_to_snapshot('testdb', 'customer_orders', 8954597067493422955)
 
+Register table
```

Comment:
Place this content before https://trino.io/docs/current/connector/iceberg.html#migrating-existing-tables

Reply:
Addressed